cmd/cloud/organization.go

```diff
@@ -54,6 +63,41 @@ func newOrganizationSwitchCmd(out io.Writer) *cobra.Command {
 	return cmd
 }
 
+func newOrganizationAuditLogs(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "audit-logs --organization-name [organization_name]",
+		Aliases: []string{"al"},
+		Short:   "Manage your organization audit logs.",
+		Long:    "Manage your organization audit logs.",
+	}
+	cmd.AddCommand(
+		newOrganizationExportAuditLogs(out),
+	)
+	cmd.PersistentFlags().StringVarP(&orgName, "organization-name", "n", "", "Name of the organization to manage audit logs for.")
+	err := cmd.MarkPersistentFlagRequired("organization-name")
+	if err != nil {
+		panic(err)
```

Comment:
```suggestion
		log.Fatalf("Error marking organization-name flag as required in astro organization audit-logs command: %s", err.Error())
```
Better to send a meaningful error to the user instead of code panic with an entire stack trace